acid/features/auth/controller.py

```diff
@@ -14,8 +14,9 @@
 
 @auth.route('/sign_in')
 def sign_in():
+    config = current_app.config
```

Comment:
extract section specific config here, like in dp.py
so:
config = current_app.config['authentication']